docs/en/integrations/migration/clickhouse-to-cloud.md

```diff
@@ -69,7 +69,7 @@ CREATE DATABASE db
 - Using the CREATE TABLE statement from the source, create the destination.
 
 :::tip
-Change the ENGINE to to ReplicatedMergeTree without any parameters when you run the CREATE statement.  ClickHouse Cloud always replicates tables and provides the correct parameters.
+Change the ENGINE to to ReplicatedMergeTree without any parameters when you run the CREATE statement. ClickHouse Cloud always replicates tables and provides the correct parameters. Keep the `ORDER BY`, `PRIMARY KEY`, and `ORDER BY` clauses though.
```

Comment:
ORDER BY is there twice, did you mean the second one to be SETTINGS or something else?

Reply:
fixed